android/sample/src/main/kotlin/com/gu/source/MainActivity.kt

```diff
@@ -36,32 +31,49 @@ class MainActivity : ComponentActivity() {
         enableEdgeToEdge()
         setContent {
             AppColourMode {
-                Greeting(name = "Android", modifier = it)
+                Greeting(modifier = it)
             }
         }
     }
 }
 
+private enum class SheetContentType {
+    Palette,
+    PagerProgressBar,
+    Buttons,
+}
+
 @SuppressLint("DiscouragedApi")
 @Composable
-private fun Greeting(name: String, modifier: Modifier = Modifier) {
+private fun Greeting(modifier: Modifier = Modifier) {
     val coroutineScope = rememberCoroutineScope()
     val scaffoldState = rememberBottomSheetScaffoldState()
 
+    var sheetContentType by remember {
+        mutableStateOf(SheetContentType.Palette)
+    }
+
     BottomSheetScaffold(
-        sheetContent = { Palette() },
+        sheetContent = {
```

Comment:
Slightly refactored the sample. Sheet content shows previews while the base scaffold just displays buttons to select different types of previews.
The sample needs more love as we add more components - that'd be done in another PR.